javascriptv3/example_code/cross-services/video-analyzer/src/js/index.js

```diff
@@ -0,0 +1,307 @@
+          finished = true;
+        }
+      }
+      finished = false;
+      // Parse results into CVS format.
+      const noOfFaces = results.Faces.length;
+      var i;
+      for (i = 0; i < results.Faces.length; i++) {
+        var boundingbox = JSON.stringify(results.Faces[i].Face.BoundingBox);
+        var confidence = JSON.stringify(results.Faces[i].Face.Confidence);
+        var pose = JSON.stringify(results.Faces[i].Face.Pose);
+        var quality = JSON.stringify(results.Faces[i].Face.Quality);
+        var arrayfirst = [];
+        var arraysecond = [];
+        var arraythird = [];
+        var arrayforth = [];
+        arrayfirst.push(boundingbox);
+        arraysecond.push(confidence);
+        arraythird.push(pose);
+        arrayforth.push(quality);
+        arrayfirst.push(arraysecond);
+        arrayfirst.push(arraythird);
+        arrayfirst.push(arrayforth);
+        facesArray.push(arrayfirst);
+      }
+      // Create the CSV file.
+      create_csv_file(facesArray);
+    } catch (err) {
+      console.log("Error", err);
+    }
+  } catch (err) {
+    console.log("Error", err);
+  }
+};
+window.ProcessImages = ProcessImages;
+
+// Helper file to create the CSV file.
+function create_csv_file(facesArray) {
+  //define the heading for each row of the data
+  var csv = "Bounding Box, , , , Confidance, Pose, , ,  Quality, ,\n";
+
+  // Merge the data with CSV.
+  facesArray.forEach(function (row) {
+    csv += row.join(",");
+    csv += "\n";
+  });
+  // Upload the CSV file to Amazon S3.
+  uploadFile(csv);
+}
+
+// Helper file to upload file to Amazon S3.
+const uploadFile = async (csv) => {
+  const uploadParams = {
+    Bucket: BUCKET,
+    Body: csv,
+    Key: "Face.csv",
+  };
+  try {
+    const data = await s3Client.send(new PutObjectCommand(uploadParams));
+    const linkToCSV =
+      "https://s3.console.aws.amazon.com/s3/object/" +
+      uploadParams.Bucket +
+      "?region=" +
+      REGION +
+      "&prefix=" +
+      uploadParams.key;
+    console.log("Success. Report uploaded to " + linkToCSV + ".");
+
+    // Send an email to notify user when report is available.
+    sendEmail(uploadParams.Bucket, uploadParams.Key);
+  } catch (err) {
+    console.log("Error", err);
+  }
+};
+// Helper function to send an email to user.
+const sendEmail = async (bucket, key) => {
+  const toEmail = document.getElementById("email").value;
+  const fromEmail = "briangermurray@gmail.com";
+  try {
+    const linkToCSV =
+      "https://s3.console.aws.amazon.com/s3/object/" +
+      bucket +
+      "?region=" +
+      REGION +
+      "&prefix=" +
+      key;
+    // Set the parameters
+    const params = {
+      Destination: {
+        /* required */
+        CcAddresses: [
+          /* more items */
+        ],
+        ToAddresses: [
+          toEmail, //RECEIVER_ADDRESS
+          /* more To-email addresses */
+        ],
+      },
+      Message: {
+        /* required */
+        Body: {
+          /* required */
```

Comment:
Is Html required? I would move the comment to inline/after Body: or Html:, to make is explicit.